Update/wata_ep_02.txt

```diff
@@ -622,7 +622,7 @@
 
 
 
-//「エエエエ、エロ本なんか持ってこないで下さい～！！＠　持って帰って持って帰って！＠　痛ててててて...！！￥  
+//「エエエエ、エロ本なんか持ってこないで下さい～！！＠　持って帰って持って帰って！＠　痛ててててて...！！￥
```

Comment:
This is important to remove because the spaces actually break the following line.